spec/ascii-art-spec.js

```diff
@@ -0,0 +1,51 @@
+/*
+ * decaffeinate suggestions:
+ * DS101: Remove unnecessary use of Array.from
+ * DS102: Remove unnecessary code created because of implicit returns
+ * Full docs: https://github.com/decaffeinate/decaffeinate/blob/master/docs/suggestions.md
+ */
```

Comment:
It looks like a couple more steps are needed to finish decaffeinating this file. ☝️

If you'd like any help working through these steps, just let me know. 🤙

Reply:
I've ran standard --fix and remove this message but would love some help on this.  Probably need to look at `ascii-art.js`  as well.  Want to setup a pairing session? @jasonrudolph